credhub-server/docker-compose.yml

```diff
@@ -1,14 +1,14 @@
 services:
   uaa:
-    image: pcfseceng/uaa
+    image: harbor-repo.vmware.com/springcloudservices/spring-credhub-uaa-server
     volumes:
       - ./uaa.yml:/uaa/uaa.yml
     ports:
       - "8080:8080"
     restart: always
 
   credhub:
-    image: ampersand8/credhub
+    image: harbor-repo.vmware.com/springcloudservices/spring-credhub-credhub-server
```

Comment:
This won't work for anyone outside of VMware. Not sure how best to handle this...

Reply:
Neither do I. Other options I considered were:
- Duplicating the docker-compose with an external registry
- Replacing these two lines when running in CI
- Hosting them somewhere else

But I was not very happy with any of those.

Reply:
We could add a registry mirror in CI and keep using the dockerhub references?

By adding 

```
{
        "registry-mirrors": ["..."]
}
```

to `/etc/docker/daemon.json`